50.IESProfileTest/028e97564391140b1476695ae7a46fa4.ies

```diff
@@ -0,0 +1,202 @@
+IESNA:LM-63-1995
+[TEST] 
+[MANUFAC] BEGA
+[MORE] Copyright LUMCat V 
+[LUMCAT] 
+[LUMINAIRE] 84658K4 (Preliminary)
+[LAMPCAT] LED  46,6W
+[LAMP]    7200 lm,51 W
+TILT=NONE
+1 -1 1.0 37 37 1 2 0.240 0.270 0.000
+1.0 1.0 51
```

Comment:
this thing is now in media folder under `mitsuba/aniso_ies`

Reply:
resolved in following commit